src/featbin/wav-copy.cc

```diff
@@ -26,35 +27,69 @@ int main(int argc, char *argv[]) {
   try {
     using namespace kaldi;
     const char *usage =
-        "Copy archives of wave files\n"
+        "Copy wave file or archives of wave files\n"
         "\n"
-        "Usage:  wav-copy [options...] <wav-rspecifier> <wav-rspecifier>\n"
+        "Usage: wav-copy [options] <wav-rspecifier> <wav-wspecifier>\n"
+        "  or:  wav-copy [options] <wav-rxfilename> [<wav-wxfilename>]\n"
         "e.g. wav-copy scp:wav.scp ark:-\n"
+        "     wav-copy wav.ark:123456 \n"
         "See also: wav-to-duration extract-segments\n";
-    
+
     ParseOptions po(usage);
 
     po.Read(argc, argv);
 
-    if (po.NumArgs() != 2) {
+    if (po.NumArgs() < 1 || po.NumArgs() > 2) {
```

Comment:
I only made the second argument optional for the new use case. I thought there are actually quite a couple of kaldi programs that allow to leave out the last arguments (source `find . -name "*.cc" | xargs grep "NumArgs() >"`), or does it have a different meaning in those cases? At least the first on the list, bin/add-self-loops.cc, has the same behaviour that a left out last argument is assumed to be stdout?

I do see that the coding style is normally a small bit different, with calling po.GetArg(n) anyway, even if n doesn't exist.